inception-project/inception
# Feature/856 855 add reification and default language to kb profiles

**What's in the PR**
(#887 is already merged into this PR to avoid conflicts. So please review and merge #887 first)
* Add reification field and default language field to kb profiles (Issues #855 & #856)

**How to test manually**
* Create a knowledge base from a profile
* See if values for default language and reification are set according to the profile

**Automatic testing**
* [X] PR includes unit tests (Changes existing tests)

**Documentation**
